Client/Clique/Common/Storage/UserStorage.swift

```diff
@@ -0,0 +1,37 @@
+import UIKit
+
+private enum DefaultValues: String {
+    
+   case email = "email"
+    case token = "token"
+    
+}
+
+final class UserStorage {
+
+static var email: String {
+    set {
+        _set(value: newValue, key: .email)
+    } get {
+        return _get(valueForKay: .email) as? String ?? ""
+    }
+}
+    
+    static var token: String{
+        set {
+            _set(value: newValue, key: .token)
+        } get {
+            return _get(valueForKay: .token) as? String ?? ""
+        }
+    }
+
+
+private static func _set(value: String, key: DefaultValues) {
```

Comment:
Da pojednostaviš implementaciju (iako malo nesigurnije) ovo može izgledati i kao;
```suggestion
private static func _set(value: Any?, key: DefaultValues) {
```